client/styles/importer.less

```diff
@@ -1,3 +1,5 @@
+@cache-version: `process.env.npm_package_version` ;
```

Comment:
Why is this an environment variable? Are we using an environment variable for the package version anywhere else?